src/components/SendFlow/Beacon/OriginationOperationSignModal.test.tsx

```diff
@@ -0,0 +1,93 @@
+import { BeaconMessageType, NetworkType, OperationRequestOutput } from "@airgap/beacon-wallet";
+import type { BatchWalletOperation } from "@taquito/taquito/dist/types/wallet/batch-operation";
+import BigNumber from "bignumber.js";
+
+import { OriginationOperationSignModal } from "./OriginationOperationSignModal";
+import { mockContractOrigination, mockImplicitAccount } from "../../../mocks/factories";
+import {
+  act,
+  dynamicModalContextMock,
+  render,
+  screen,
+  userEvent,
+  waitFor,
+} from "../../../mocks/testUtils";
+import { GHOSTNET, MAINNET } from "../../../types/Network";
+import { WalletClient } from "../../../utils/beacon/WalletClient";
+import { prettyTezAmount } from "../../../utils/format";
+import { useGetSecretKey } from "../../../utils/hooks/getAccountDataHooks";
+import { networksActions } from "../../../utils/redux/slices/networks";
+import { store } from "../../../utils/redux/store";
+import { executeOperations, makeToolkit } from "../../../utils/tezos";
+import { SuccessStep } from "../SuccessStep";
+
+const user = userEvent.setup();
```

Comment:
it's better to do inside the tests according to [this](https://testing-library.com/docs/user-event/intro/#writing-tests-with-userevent)